src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/organizations.py

```diff
@@ -162,6 +172,15 @@ def describe_ou_name(self, ou_id):
         except ClientError as error:
             raise RootOUIDError("OU is the Root of the Organization") from error
 
+    def describe_account_name(self, account_id):
```

Comment:
Done. I didn't want to add an `import datetime`, so I excluded the JoinedTimestamp result from the describe_account mock operation instead, since we aren't using that field right now. But I can add the import if you'd prefer.